concent-vm/start-docker-services.yml

```diff
@@ -0,0 +1,37 @@
+- name:   Check if rabbitmq container is running
+  command:       docker inspect --format {% raw %}"{{.State.Running}}"{% endraw %} rabbitmq
```

Comment:
Please align `name` here and in the other playbooks.

Reply:
Well, do you want to align the `name` directive in all playbooks now? In this pull request ?

Reply:
At least in the newly created playbooks.

Reply:
Done.

Reply:
Not done yet.

I mean that this:
``` yaml
- name:   Check if geth container is running
  command:       docker inspect --format {% raw %}"{{.State.Running}}"{% endraw %} geth
  register:      geth_container_running
  ignore_errors: True
```

should look like this:
``` yaml
- name:          Check if geth container is running
  command:       docker inspect --format {% raw %}"{{.State.Running}}"{% endraw %} geth
  register:      geth_container_running
  ignore_errors: True
```

Reply:
Done.